compressor/storage.py

```diff
@@ -4,12 +4,25 @@
 import time
 from urllib.parse import urljoin
 
-from django.core.files.storage import FileSystemStorage, get_storage_class
+from django.core.files.storage import FileSystemStorage, InvalidStorageError, storages
 from django.utils.functional import LazyObject, SimpleLazyObject
 
 from compressor.conf import settings
 
 
+def get_storage(
+    alias=settings.COMPRESS_STORAGE_ALIAS,
+    storage_class=settings.COMPRESS_STORAGE
+):
+    try:
+        return storages[alias]
+    except InvalidStorageError:
+        storages.backends[alias] = {
+            "BACKEND": storage_class
+        }
```

Comment:
Modifying `django.core.files.storage.storages` ? That feels kinda wrong....

Reply:
I plan to request that this be made public API, just so that we have the stability guarantee.

Reply:
Forum thread for that: https://forum.djangoproject.com/t/document-storagehandler-as-public-api/32101

Reply:
Even if `storages` dict ends up being documented it feels weird to me to modify it in an app, but I don't have a better API suggestion and I trust your judgment on that.

Reply:
I'm open to another way of tying into the new API but other than forcing users to declare the storages we need in settings, which will almost never be customised, I don't see it. 🤔

"The StorageHandler is instantiated from settings, and then apps can add defaults to it if needed", is the story I'm going for.

Reply:
Opened a PR for that: https://github.com/django/django/pull/18277